### PR TITLE
Configure `gettext` to generate a `pot` file for each `rst` file.

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
Depends on:

- [x] #5642
- [x] #5636
- [x] #5645
- [x] #5646

### CODE REVIEW

File: docs/locale/fr/LC_MESSAGES/adminguide/hosting/index.po
Comment (FirelightFlagboy) [inline docs/locale/fr/LC_MESSAGES/adminguide/hosting/index.po:1]: `adminguide.po` wasn't renamed to this file.

This translation file is for the `on-premise hosting`'s page.

File: docs/locale/fr/LC_MESSAGES/adminguide/index.po [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-present, Scille SAS
+# This file is distributed under the same license as the Parsec package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
```
Comment (mmmarcos) [inline docs/locale/fr/LC_MESSAGES/adminguide/index.po:4]: This does not come from a template? How about putting something like these in all files:
```suggestion
# Copyright (C) 2016-present, Scille SAS
# This file is distributed under the same license as the Parsec package.
# Maintained by the parsec-cloud team. 
# https://parsec.cloud/
```

**Review by mmmarcos — APPROVED** [review]
LGTM

File: docs/locale/fr/LC_MESSAGES/adminguide/index.po (hunk 1 above)
Comment (FirelightFlagboy) [reply docs/locale/fr/LC_MESSAGES/adminguide/index.po]: done

**Pull request merged**